src/components/ProfileList/ProfileList.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react'
+import styles from '@/components/ProfileList/ProfileList.module.scss'
+import ProfilePicture from '@/assets/images/ProfilePicture.png'
+import { ProfileData } from '@/components/ProfileList/ProfileData'
+interface ProfileListItem {
+  id: number
+  name: string
+  seniority: string
+  location: string
+  technology: string
+  technology2: string
+  technology3: string
+  others: string
+}
+
+interface ProfileListProps {
+  data: ProfileListItem
+}
+
+enum JobSpecialization {
+  Frontend = 'Frontend',
+  Backend = 'Backend',
+  Fullstack = 'Fullstack',
+}
+
+interface TechnologyChipProps {
+  name: string
+  theme: JobSpecialization
+  seniority: string
+}
+
+const getTechnologyClass = (theme: JobSpecialization, seniority: string) => {
+  if (seniority.includes('Frontend')) {
+    return styles.frontend
+  } else if (seniority.includes('Backend')) {
+    return styles.backend
```

Comment:
**nitpick:** We should split it into 2 separate functions, like: `getTechnologyClassByTheme`, and `getTechnologyClassBySeniority`, so they handle one argument